semgrep_output_v1.atd

```diff
@@ -919,6 +929,55 @@ type scan_request = {
     ?scan_metadata: scan_metadata option;
 }
 
+(* Response by the backend to the CLI to the POST /scans *)
+(* TODO
+type scan_response = {
+    scan: scan_response_bis;
+}
+type scan_response_bis = {
+    (* the scan id *)
+    id: string;
+    (* since 1.37.0 *)
+    ?enabled_products: product list;
+}
+*)
+
+(* TODO? seems used only for dry_run and passed urlencoded to /scans/config?xxx
```

Comment:
Note to dig into this: I thought this re-used the same data as the scan create payload

Reply:
Yes it's weird we have such a complex structure that we just use for a dry_run ...